frontend-web/webclient/app/Files/LowLevelFileTable.tsx

```diff
@@ -575,6 +666,22 @@ export const LowLevelFileTable: React.FunctionComponent<LowLevelFileTableProps>
                             />
                         </StickyBox>
                     )}
+                    <ReactModal isOpen={selectProduct}
+                        onRequestClose={() => setSelectProduct(false)}
+                        shouldCloseOnEsc
+                        ariaHideApp={false}
```

Comment:
I think this messes with screen-readers. Adding the other approach mention by the `console.log` instead.

Reply:
You haven't removed this line?